src/routes/delegates/delegates.service.ts

```diff
@@ -73,4 +74,17 @@ export class DelegatesService {
       createDelegateDto.label,
     );
   }
+
+  async deleteDelegate(
+    chainId: string,
+    delegateAddress: string,
+    deleteDelegateDto: DeleteDelegateDto,
+  ): Promise<unknown> {
+    return await this.repository.deleteDelegate(
```

Comment:
As we talked offline it seems `delegateAddress` is sent twice by the client (as path param and as part of the body).  As you mentioned @moisses89 it seems the Transaction Service overrides the content of the body with the `delegateAddress` value in the path ([see this](https://github.com/safe-global/safe-transaction-service/blob/0ac2f06b4e2c63e039a60f99fcb4cf2ffd91ac8f/safe_transaction_service/history/views.py#L851)) so I know we have to focus in feature-parity with the current CGW, but is this maybe a bit misleading for clients? Should we check the path and the body values match? cc. @Uxio0 @fmrsabino

Reply:
I think that's something that I started doing with the `safe address` and I extended that to the delegates. There are a lot of places in the **tx service** where `safe address` can be part of the body but it will be derived from the `url`, and I did the same for the delegate address

Reply:
https://github.com/safe-global/safe-transaction-service/pull/1135